msf_updates/src/position_msf/init_yaw.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+PACKAGE = 'msf_updates'
+import roslib;roslib.load_manifest(PACKAGE)
+import rospy
+from geometry_msgs.msg import Vector3, Quaternion
+from std_msgs.msg import Float64
+import tf
+from math import *
+
+import dynamic_reconfigure.client
+
+client = dynamic_reconfigure.client.Client(rospy.get_param("msf_sensor_node",
```

Comment:
A command line argument is better suited for this, so that the script is kept stand-alone. But it's OK like this too. Good job.